modules/installation-launching-installer.adoc

```diff
@@ -327,26 +319,21 @@ https://ovirtlab.example.com/ovirt-engine/api
 ----
 endif::openshift-origin[]
 +
-.. For `Is the oVirt CA trusted locally?`, enter `Yes` since you have already set up a CA certificate. Otherwise, enter `No`.
-.. For `oVirt's CA bundle`, if you entered `Yes` for the preceding question, copy the certificate content from `/etc/pki/ca-trust/source/anchors/ca.pem` and paste it here. Then, press `Enter` twice. Otherwise, if you entered `No` for the preceding question, this question does not appear.
-.. For `oVirt engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
+.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to Engine?`, answer `y` or `N`. If you answer `N`, you must install {product-title} in insecure mode.
+//TODO: Add this sentence with xref after it's OK to add xrefs: For information about insecure mode, see xref:installing-rhv-insecure-mode_installing-rhv-default[].
+.. For `Engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
 +
 [source,terminal]
 ----
 <username>@<profile> <1>
 ----
 +
-<1> For `<username>`, specify the user name of an {rh-virtualization} administrator. For `<profile>`, specify the login profile, which you can get by going to the {rh-virtualization} Administration Portal login page and reviewing the *Profile* dropdown list. Together, the user name and profile should look similar to this example:
-+
-[source,terminal]
-----
-admin@internal
-----
+<1> For `<username>`, specify the user name of an {rh-virtualization} administrator. For `<profile>`, specify the login profile, which you can get by going to the {rh-virtualization} Administration Portal login page and reviewing the *Profile* dropdown list. For example: `admin@internal`
```

Comment:
I think you need a period at the end here still even though it's not a full sentence.

```
For example: `admin@internal`.
```